### PR TITLE
Add instructions for putting images onto the ARP

### PR DESCRIPTION
**Pull request by DunderRoffe**
Add instructions for putting PELUX images on to the ARP. This includes the
booting from USB, SD-card and EMMC.

The instructions focus on the ARP as an intel target so some information.
Meaning that if PELUX at a later stage decides to support other SMARC
modules, such as ARM based SMARC modules, then additional information
might need to be added.

Signed-off-by: Viktor Sjölind <vsjolind@luxoft.com>

### CODE REVIEW

**Comment by DunderRoffe** [comment]
@TinyPush please verify that I am not lying about the SD-card part

**Review by JEderonn — APPROVED** [review]

File: docs/chapters/baseplatform/deploying-the-image.rst [hunk 1]
```diff
@@ -3,5 +3,5 @@
 
 Image names for PELUX targets
 -----------------------------
-* NUC/Minnowboard: ``core-image-pelux-intel-corei7-64.wic``
+* NUC, Minnowboard and ARP with intel SMARC: ``core-image-pelux-intel-corei7-64.wic``
```
Comment (sashko) [inline docs/chapters/baseplatform/deploying-the-image.rst]: Intel. Do we really support Minnowboard?

File: docs/chapters/baseplatform/booting-on-target-hardware.rst [hunk 2]
```diff
@@ -2,6 +2,21 @@ Booting on target hardware
 ==========================
 .. note:: Login as user ``root``.
 
+ARP
+---
+
+On the ARP there are three options available for booting.
```
Comment (sashko) [inline docs/chapters/baseplatform/booting-on-target-hardware.rst]: :

File: docs/chapters/baseplatform/booting-on-target-hardware.rst [hunk 3]
```diff
@@ -2,6 +2,21 @@ Booting on target hardware
...
+
+1. USB: Boot from a USB-stick by simply inserting said USB-stick into one of the
+   USB ports on the arp. Make sure that USB is set as boot source in bios/u-boot
```
Comment (sashko) [inline docs/chapters/baseplatform/booting-on-target-hardware.rst]: ARP, BIOS

File: docs/chapters/baseplatform/booting-on-target-hardware.rst [hunk 4]
```diff
@@ -2,6 +2,21 @@ Booting on target hardware
...
+2. SD-card: Put insert the SD-card into the slot labeled "SD card" located next
```
Comment (sashko) [inline docs/chapters/baseplatform/booting-on-target-hardware.rst]: s/put/insert/

File: docs/chapters/baseplatform/booting-on-target-hardware.rst [hunk 5]
```diff
@@ -2,6 +2,21 @@ Booting on target hardware
...
+   to the USB ports. This feature requires ARP version >= 1.5.
+3. EMMC: On most SMARC devices there is an EMMC which can be used for booting.
+   This is the boot option with best performance. However, it requires some
+   setup to get the PELUX image on to the EMMC. An example of such a setup would
+   be to put the image on a USB-stick with Ubuntu-live and then from there use
```
Comment (sashko) [inline docs/chapters/baseplatform/booting-on-target-hardware.rst]: s/to put/putting/, s/the image/an image/, s/use/using/

File: docs/chapters/baseplatform/booting-on-target-hardware.rst (hunk 4 above)
Comment (TinyPush) [inline docs/chapters/baseplatform/booting-on-target-hardware.rst]: Slot is labeled "SMARC SD".

File: docs/chapters/baseplatform/deploying-the-image.rst (hunk 1 above)
Comment (DunderRoffe) [reply docs/chapters/baseplatform/deploying-the-image.rst]: I do not know but I do not want to change that in this commit

File: docs/chapters/baseplatform/booting-on-target-hardware.rst (hunk 4 above)
Comment (sashko) [inline docs/chapters/baseplatform/booting-on-target-hardware.rst]: so... do you.... like... put... or... insert..?

File: docs/chapters/baseplatform/booting-on-target-hardware.rst [hunk 6]
```diff
@@ -2,6 +2,21 @@ Booting on target hardware
...
+   `dd` to write the image to the EMMC.
```
Comment (sashko) [inline docs/chapters/baseplatform/booting-on-target-hardware.rst]: onto EMMC? my English is no good though.

File: docs/chapters/baseplatform/deploying-the-image.rst (hunk 1 above)
Comment (sashko) [reply docs/chapters/baseplatform/deploying-the-image.rst]: Intel

File: docs/chapters/baseplatform/booting-on-target-hardware.rst (hunk 4 above)
Comment (DunderRoffe) [reply docs/chapters/baseplatform/booting-on-target-hardware.rst]: Ok so this time I actually added my changes :angel:

**Review by sashko — APPROVED** [review]

**Review by thenor — APPROVED** [review]

**Pull request merged**